portfolio/src/main/webapp/script.js

```diff
@@ -122,6 +122,16 @@ function createListElement(text) {
   return liElement;
 }
 
+function updateContactButton() {  // eslint-disable-line no-unused-vars
+  if (document.getElementById('name').value === '' ||
+      document.getElementById('message').value === '' ||
+      document.getElementById('email').value === '') {
```

Comment:
Should we consider the cases where values contain only spaces or tabs?

Reply:
Yea, that would be a good idea. I will add that.